__tests__/js/Relude_Js_Json_test.re

```diff
@@ -792,7 +792,7 @@ describe("Json", () => {
     let actual =
       MyType.make
       <$> (JD.stringAt(0, json) >>= (a => VOk(a ++ a)))  // bind (flatMap) a value here just for fun
-      <*> (JD.intAt(1, json) <#> (a => a * 2))  // map a value here - <#> is flipMap - we need to flip it b/c the Validation comes first here
+      <*> (JD.intAt(1, json) <$$> (a => a * 2))  // map a value here - <#> is flipMap - we need to flip it b/c the Validation comes first here
```

Comment:
Probably want to change the `<#>` in the comment here as well.

Reply:
great point. I thought I'd gotten all of them!